src/main/java/com/google/javascript/gents/TypeAnnotationPass.java

```diff
@@ -334,6 +339,14 @@ private void setTypeExpression(Node n, @Nullable JSTypeExpression type, boolean
     }
   }
 
+  /** Sets the annotated type expression corresponding to Node {@code n}. */
+  private void setTypeExpression(Node n, @Nullable TypeDeclarationNode type) {
```

Comment:
Is this really useful, despite being 5 lines it does copy/paste the one above, so it can be a maintenance burden.

Reply:
My first attempt here was to convert the TypeDeclarationNode I created back into a JSTypeExpression before feeding it into the existing function, and that didn't work, so I do think this is necessary.  I DRY'd it up by having the old one call into this one instead of repeating.

Reply:
that's better.